spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/DeleteOrphanFilesSparkAction.java

```diff
@@ -234,28 +241,137 @@ private String jobDesc() {
     return String.format("Deleting orphan files (%s) from %s", optionsAsString, table.name());
   }
 
-  private void deleteFiles(SupportsBulkOperations io, List<String> paths) {
-    try {
-      io.deleteFiles(paths);
-      LOG.info("Deleted {} files using bulk deletes", paths.size());
-    } catch (BulkDeletionFailureException e) {
-      int deletedFilesCount = paths.size() - e.numberFailedObjects();
-      LOG.warn(
-          "Deleted only {} of {} files using bulk deletes", deletedFilesCount, paths.size(), e);
-    }
+  private boolean streamResults() {
```

Comment:
Is this worth it's own method? Only used once

Reply:
There are some places where we handle differently, like: https://github.com/apache/iceberg/blob/f7e6a2775bd5a9e202465e6281c56d8c41fd7f0e/core/src/main/java/org/apache/iceberg/RemoveSnapshots.java#L90-L92

I see your point. Still a bit strange for me, but I don't have strong opinion on this